.github/workflows/codeql-analysis.yml

```diff
@@ -20,16 +20,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ 'javascript' ]
+        language: ['javascript']
 
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@v4
+      - name: Checkout repository
+        uses: actions/checkout@v4
```

Comment:
Are you sure this indentation isn't meaningful? This kind of whitespace stuff is what always scares me when working with YAML!

Reply:
this is a prettier fix! i confirmed with the github actions VSCode extension that this is still valid 👍🏽

_(but i agree, fuckin yaml...)_